docs/src/main/asciidoc/security-keycloak-authorization.adoc

```diff
@@ -318,7 +318,7 @@ public class ProtectedResource {
     @GET
     public CompletionStage<List<Permission>> get() {
         return identity.checkPermission(new AuthPermission("{resource_name}"))
-                .thenCompose(granted -> {
+                .transform(granted -> {
```

Comment:
Missing `onItem()` before calling `.transform(`.

Reply:
Going to update it shortly